packages/fxa-settings/src/constants/index.tsx

```diff
@@ -7,3 +7,8 @@ export const HomePath = '/settings';
 export const DeleteAccountPath = '/settings/delete_account';
 export const VPNLink = 'https://vpn.mozilla.org/';
 export const MonitorLink = 'https://monitor.firefox.com/';
+export const SyncLink = 'https://www.mozilla.org/firefox/sync/';
+export const RelayLink = 'https://relay.firefox.com';
+export const AddonsLink = 'https://addons.mozilla.org';
+export const MDNLink = 'https://developer.mozilla.org';
+export const HubsLink = 'https://hubs.mozilla.com';
```

Comment:
nit: Existing `VPNLink` and `MonitorLink` URIs have trailing `/`, but not the newer `RelayLink`, `AddonsLink`, `MDNLink`, and `HubsLink`.
Probably not relevant or interesting unless we're appending paths to these or something which might get tricky.

Reply:
Ah yes, well spotted, thank you. Updated.